### DISCUSSION TITLE
There is AWS Known issue of : AWS Network Firewall egress filtering can be easily bypassed

### DISCUSSION THREAD

**Discussion by ved-ant-jain** [globaldatanet/aws-firewall-factory#440] [Q&A] [answered]
Hi there,

I recently came across a known issue where AWS Network Firewall egress filtering can be easily bypassed, as detailed in this [blog post](https://canglad.com/blog/2023/aws-network-firewall-egress-filtering-can-be-easily-bypassed/). 

Could you please clarify if your tool is capable of addressing or mitigating this issue? Any insight would be greatly appreciated!

**Comment by daknhh** [comment, marked as answer]
Hi, right now AWS Network Firewall is not implemented yet. We are just supporting AWS WAF and Advanced Shield.

**Reply by ved-ant-jain** [reply]
Gotcha, Is there any plans you guys will look this issue in AWS and try to solve it ?

**Reply by daknhh** [reply]
We are planning to add Networking Firewalls support also to AWS Firewall Factory, yes.

**Reply by daknhh** [reply]
And just a side not we are not directly from AWS we are just some a OpenSource which help you to provision firewalls on AWS
